tests/Aspire.Hosting.Tests/Qdrant/AddQdrantTests.cs

```diff
@@ -0,0 +1,220 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We should also have component tests. See the other components for examples.

It doesn't look like there is a Qdrant test container (yet). https://github.com/testcontainers/testcontainers-dotnet/issues/992. But we can make the tests before that, and just have them check if a connection can be made and skip if it can't. Then you can run the tests with a local Qdrant container running.

Reply:
Yeah, FYI I was in touch recently with @russcam about reviving that PR, hopefully he'll have a bit of time soon.

Reply:
I've updated the PR and requested the maintainers to reopen it (I don't have ability to reopen it- guessing there's some repo setting preventing this).